client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -113,16 +111,15 @@ const Sidebar = ( { sidebarDomain, siteSlug, submit, goToStep, flow }: SidebarPr
 			submit,
 			globalStylesInUse && shouldLimitGlobalStyles,
 			PLAN_PREMIUM,
-			setShowPlansModal,
 			queryClient,
-			goToStep,
 			flow,
 			isEmailVerified,
 			checklistStatuses,
+			() => setShowConfirmModal( true ),
+			goToStep,
```

Comment:
See this comment about change in order for props here: https://github.com/Automattic/wp-calypso/pull/82778/files#r1350784482